.github/workflows/scraping.yaml

```diff
@@ -0,0 +1,51 @@
+name: Scraping the repositories from Source Graph
```

Comment:
This will consume all the requests I can do to GitHub, right?

Reply:
This will interact with SourceGraph - it will be consuming events from the SSE endpoint. SourceGraph does not require any token to use their search engine, however, the result set is not that big either - about 530 repos. Maybe we could investigate whether we could use github search as an additional source of data. Or maybe SourceGraph provides API tokens which could lead to better search results

Reply:
no no, it's perfect then!